app/Http/Controllers/Common/TemplateController.php

```diff
@@ -323,7 +323,7 @@ public function prices($id)
                 $currency = $currencyAndSymbol['currency'];
                 $planData = $currencyAndSymbol['plan'];
 
-                if (! $planData || ($planData->add_price ?? 0) <= 0) {
+                if (! $planData || is_null($planData->add_price ?? null)) {
```

Comment:
add_price Change the data type of this column. Use it as an integer or float and set the default value to 0.